livekit-ffi/src/server/room.rs

```diff
@@ -116,8 +116,24 @@ impl FfiRoom {
     ) -> proto::ConnectResponse {
         let async_id = server.next_id();
 
+        let sdk;
+        let sdk_version;
+        {
+            let config = server.config.lock();
+            sdk = config.as_ref().map(|c| c.sdk.clone()).unwrap();
+            sdk_version = config.as_ref().map(|c| c.sdk_version.clone()).unwrap();
+        }
+
```

Comment:
```suggestion
        let (sdk, sdk_version) = {
            let config = server.config.lock();
            (config.as_ref().map(|c| c.sdk.clone()).unwrap(), config.as_ref().map(|c| c.sdk_version.clone()).unwrap())
        }

```

Reply:
let's verify they're not empty instead of unrwap?

Reply:
yeah i reworked this and it turned out they would be non-empty, the problem was simply whether the config existed or not. i redid it to be more clear and it won't use them unless the config is non-empty